mk/spksrc.download.mk

```diff
@@ -157,7 +157,7 @@ download_target: $(PRE_DOWNLOAD_TARGET)
 	      else \
 	        $(MSG) "  wget $${url}" ; \
 	        rm -f $${localFile}.part ; \
-	        wget --secure-protocol=TLSv1_2 -nv -O $${localFile}.part -nc $${url} ; \
+	        wget --secure-protocol=TLSv1_2 --no-check-certificate -nv -O $${localFile}.part -nc $${url} ; \
```

Comment:
Please no.

Reply:
Agreed!!

Reply:
good call, and no worry, this is **temporary** as there was a download failure due to a invalid certificate for `native/yasm` thus blocking the build.

Reply:
Ok thanks for the additional info, I mean you got to do what you got to do (for testing). But I really don't want it anywhere close to the main branch.

Reply:
Is it https://www.tortall.net/ with the invalid cert?

Reply:
It's probably better to switch to a GitHub release then: https://github.com/yasm/yasm/releases

Reply:
I opened an issue just in case: https://github.com/yasm/yasm/issues/190

Reply:
This seems to be a recurring problem for them, so they suggest to use the GitHub release: https://github.com/yasm/yasm/issues/182#issuecomment-895487445

Reply:
Can someone tackle this (will probably offline for a few days)?  I'll rebase this PR afterwards.